Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,24 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
```

Comment:
[btw- Love this section. It's a great idea.] WRT the language: Although not strictly prohibited, a couple of the phrases may read a bit smoother if a preposition is always given an object or the sentence is restructured to eliminate the need for the preposition. Also, "what" (I think as a pronoun here) can be a little awkward. I suggest going from (and incorporating @SamuelEnglard's suggestion for using ".NET" either before or after "languages") ...

> IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.

... to ...

> Higher level [.NET] languages [in .NET], such as C#, compile down a stack based instruction set, which is called Intermediate Language (IL).

... or ...

> IL is a stack-based instruction set. Higher level [.NET] languages [in .NET], such as C#, compile down to IL.